Tests/Microsoft.AppCenter.Test.UWP/UWPPushTest.cs

```diff
@@ -153,5 +170,33 @@ public void ParseAppCenterPushMobileCenter()
             Assert.AreEqual(expectedResult.CustomData["key1"], actualResult.CustomData["key1"]);
             Assert.AreEqual(expectedResult.CustomData["key2"], actualResult.CustomData["key2"]);
         }
+
+        /// <summary>
+        /// Verify Push Service can be enabled or disabled correctly.  
+        /// </summary>
+        [TestMethod]
+        public void EnabledPushService()
+        {
+            Push.Push.SetEnabledAsync(false).Wait();
+            Assert.IsFalse(Push.Push.IsEnabledAsync().Result);
+
+            Push.Push.SetEnabledAsync(true).Wait();
+            Assert.IsTrue(Push.Push.IsEnabledAsync().Result);
+        }
+
+        /// <summary>
+        /// Verify OnUserIdUpdated works with userId changes. 
+        /// </summary>
+        [TestMethod]
+        public void OnUserIdUpdated()
```

Comment:
Should we name it as VerifyEnqueueAsyncIsCalledOnceOnUserIdUpdatedInvocation. Ideally test name should convey the intention.